astro/src/content/docs/get-started/core-concepts/collected-metrics.mdx

```diff
@@ -0,0 +1,535 @@
+---
+title: Collected Metrics
+description: An overview of how, when, and why FusionAuth collects deployment usage data.
+section: get started
+subcategory: download and install
+---
+
+One of our goals is constant improvement. In order to get there, we collect some usage data from FusionAuth deployments. This data helps us understand how you're using FusionAuth so that we can give you better features, tightened security, and tailored experiences.
```

Comment:
```suggestion
We college usage data from FusionAuth deployments to improve our product. This data helps us understand how you're using FusionAuth so that we can give you better features, tightened security, and tailored experiences.
```